src/intersection-observer-polyfill.js

```diff
@@ -33,6 +34,133 @@ import {layoutRectLtwh, rectIntersection, moveLayoutRect} from './layout-rect';
  */
 export let DOMRect;
 
+export const AMP_DEFAULT_THRESHOLD =
+    [0, 0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4,
+    0.45, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95, 1];
+
+/**
+ * A function to get the element's current IntersectionObserverEntry
+ * regardless of the intersetion ratio. Only available when element is not
+ * nested in a container iframe.
+ * @param {!./layout-rect.LayoutRectDef} element element's rect
+ * @param {?./layout-rect.LayoutRectDef} owner element's owner rect
+ * @param {!./layout-rect.LayoutRectDef} hostViewport hostViewport's rect
+ */
+export function getIntersectionChangeEntry(
```

Comment:
Almost, both `element` and `root` are significant for the called functions. We could make `opt_iframe` and `owner` generic "extras", though.